src/main/java/uk/gov/defra/datareturns/validation/activities/ActivityValidator.java

```diff
@@ -79,17 +82,30 @@ private boolean checkUniqueRiverPerSubmission(final Activity activity, final Con
     }
 
     /**
-     * Check that the count of days spent on the river is within possible maximums
+     * Check that the count of days spent on the river is within possible maximums.
+     * Zero is permitted for the FMT user (Paper submissions)
      *
      * @param activity the activity to be validated
      * @param context  the validator context
      * @return true if valid, false otherwise
      */
     private boolean checkDays(final Activity activity, final ConstraintValidatorContext context) {
         boolean valid = checkDaysMandatoryRelease(activity, context) && checkDaysOther(activity, context);
-        if (valid && activity.getDaysFishedWithMandatoryRelease() < 1 && activity.getDaysFishedOther() < 1) {
-            valid = handleError(context, "DAYS_FISHED_NOT_GREATER_THAN_ZERO", ConstraintValidatorContext.ConstraintViolationBuilder::addBeanNode);
+
+        final Optional<Submission> sub = Optional.ofNullable(activity.getSubmission());
+
+        if (sub.isPresent()) {
```

Comment:
It is not possible to have an Activity without as associated Submission - no need for the use of Optional here